web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -1381,47 +1380,12 @@ function joinup_core_module_implements_alter(array &$implementations, $hook) {
  * Implements hook_cron().
  */
 function joinup_core_cron() {
+  // By adding this line in settings.php and rebuilding the cache, the process
+  // of cached computed fields can be suppressed:
+  // @code
+  // $settings['joinup.parse_cached_computed_field'] = FALSE;
+  // @endcode
   if (Settings::get('joinup.parse_cached_computed_field', TRUE)) {
     cached_computed_field_cron();
   }
 }
-
-/**
- * Implements hook_ENTITY_TYPE_update().
- */
-function joinup_core_node_update(NodeInterface $node) {
-  /** @var \Drupal\node\NodeInterface $original */
-  $original = $node->original;
-
-  $log = function () use ($node, $original): bool {
```

Comment:
Is this debugging code that somehow ended up in the develop branch?

Reply:
Yes. As we've been discussed Friday in the standup, we revert that in the scope of this issue. However, we've decided to keep the switch for cached computed field. So I needed first to bring the code in `develop`.